content/Engineering/Networking/RemDatabases.md

```diff
@@ -0,0 +1,49 @@
+TODO] Start with comparison to spreadsheet, explaining ‘NoSQL’ Vs ‘SQL’ and that this is Relational vs Non-Relational
+
+
+As a programmer that began her career in DevOps and migrated to programming proper via OOP and databases, I feel like I may be a touch helpful here.
+
+I would also like to say that it is indeed overused and, as much as a I enjoy SQL, it consumes an absurd amount of bandwidth for what it is(functional and efficient *where and when* it is called for) and what it is not (an all-in-one database solution). SQL should be learned because it is ubiquitous and (opinion) not because it is the "the best". Oracle has, for a long time, been worthy of criticism, especially with it all but being slapped down with Anti-Trust Law violation [probably good place for a link/blog discussing monopolies in Tech, Adobe, CISCO, Oracle, MSFT, etc…]
```

Comment:
I'm sure you noticed the rant, https://opguides.info/engineering/networking/databases/#sql - I will probably better word that myself, but as `RemDatabases.md` should be merged into that page (`databases.md`) when you're done, you might want to just move that to be under here, and I can make it less rant-y and wall-of-text-y when you're done.

Reply:
Sure, that sounds good, if you want to keep it! It just kind of spilled out after I read your comments under #sql and was debating about including it or not.